test/create_db.sh

```diff
@@ -3,15 +3,47 @@ set -o errexit
 cd $(dirname $0)/..
 source test/db-common.sh
 
-# set db connection for if running in a separate container or not
+
+# set db connection for if running in a seperate container or not
 dbconn="-u root"
 if [[ ! -z "$MYSQL_CONTAINER" ]]; then
 	dbconn="-u root -h 127.0.0.1 --port 3306"
 fi
 
+
+DB_VERSION=$(  mysql $dbconn -e "status;" | grep "Server version" | sed -e 's/-Maria.*//
```

Comment:
I Just got back after being out of email/phone range for 4 days - I will
change the pull request and re-test tonight.

On Mon, Jan 11, 2016 at 6:48 PM, Jacob Hoffman-Andrews <
notifications@github.com> wrote:

> In test/create_db.sh
> <https://github.com/letsencrypt/boulder/pull/1345#discussion_r49406747>:
>
> >  dbconn="-u root"
> >  if [[ ! -z "$MYSQL_CONTAINER" ]]; then
> >  	dbconn="-u root -h 127.0.0.1 --port 3306"
> >  fi
> >
> > +
> > +DB_VERSION=$(  mysql $dbconn -e "status;" | grep "Server version" | sed -e 's/-Maria.*//
>
> I think this is fancier and more future-proof than we would like to
> maintain. How about:
>
> DB_VERSION=$( mysql $dbconnect -e "status;" | grep "Server version")
>
> if [[ $DB_VERSION =~ "10.1" ]] ; then
>   ... use new sql file
> elif [[ $DB_VERSION =~ "10.0" ]] ; then
>   ... use old sql file
> else
>   echo "DB version $DB_VERSION not supported."
>   exit
> fi
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/letsencrypt/boulder/pull/1345/files#r49406747>.
>



-- 
Philip Schlump